src/app/shared/header/header.component.html

```diff
@@ -9,7 +9,7 @@
   </mat-form-field>
   <span class="spacer"></span>
   <ul>
-    <span *ngIf="accountService.isUserLoggedIn()">
+    <span *ngIf="accountService.isUserLoggedIn(); else signUpOption">
```

Comment:
Does the else statement call the below template? That's cool. I've never seen that syntax. We'll have to talk about this later so I can start implementing that in other parts of the code.

Reply:
yup! the beauty of ng-template!